src/recommendations/RecommendationsPage.jsx

```diff
@@ -13,19 +13,19 @@ import {
 import PropTypes from 'prop-types';
 import { Helmet } from 'react-helmet';
 
-import { EDUCATION_LEVEL_MAPPING, POPULAR } from './data/constants';
+import { EDUCATION_LEVEL_MAPPING } from './data/constants';
 import useRecommendations from './data/hooks/useRecommendations';
 import messages from './messages';
 import RecommendationsLargeLayout from './RecommendationsPageLayouts/LargeLayout';
 import RecommendationsSmallLayout from './RecommendationsPageLayouts/SmallLayout';
-import { trackRecommendationsViewed } from './track';
+import { trackSkipButtonClicked } from './track';
 import { DEFAULT_REDIRECT_URL } from '../data/constants';
 
 const RecommendationsPage = ({ location }) => {
   const { formatMessage } = useIntl();
   const registrationResponse = location.state?.registrationResult;
   const educationLevel = EDUCATION_LEVEL_MAPPING[location.state?.educationLevel];
-  const userId = location.state?.userId;
+  const userId = location.state?.userId || 1234;
```

Comment:
Why do we need to set the default value here for `userId`?

Reply:
Sorry, this was added for debugging.